kasten/k10-minimum-retention/patched.yaml

```diff
@@ -0,0 +1,41 @@
+kind: Policy
+apiVersion: config.kio.kasten.io/v1alpha1
+metadata:
+  name: hourly-policy
+  namespace: kasten-io
+  labels:
+    appPriority: Mission-Critical
+spec:
+  comment: My sample custom backup policy
+  frequency: "@hourly"
+  subFrequency:
+    minutes:
+      - 30
+    hours:
+      - 22
+      - 7
+    weekdays:
+      - 5
+    days:
+      - 15
+  retention:
+    hourly: 24
+    daily: 30
+    weekly: 4
+    monthly: 6
```

Comment:
In a previous commit, you had changed `monthly` to `3` except you did not update the patched resource accordingly. As a result, this test is now failing. Would you address, please?